src/v/model/metadata.h

```diff
@@ -59,20 +60,22 @@ using initial_revision_id
 using rack_id = named_type<ss::sstring, struct rack_id_model_type>;
 struct broker_properties
   : serde::
-      envelope<broker_properties, serde::version<0>, serde::compat_version<0>> {
+      envelope<broker_properties, serde::version<1>, serde::compat_version<0>> {
     uint32_t cores;
     uint32_t available_memory_gb;
     uint32_t available_disk_gb;
     std::vector<ss::sstring> mount_paths;
     // key=value properties in /etc/redpanda/machine_properties.yaml
     std::unordered_map<ss::sstring, ss::sstring> etc_props;
+    uint64_t available_memory_bytes;
 
     bool operator==(const broker_properties& other) const {
```

Comment:
seems like we should use `= default` for this (this wasn't available when first written, so this is an opportunity that presents itself from time to time